_posts/2016-10-26-jwt-artsy-journey.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "JSON Web Tokens: Artsy's Journey"
+date: 2016-10-26 11:03
+comments: true
+author: ashkan
+categories: [Authentication, JWT, API]
+---
+
+At Artsy we currently have ~1730 client applications hitting our API and requesting authentication. When a user successfully authenticates through one of these clients, we want to embed basic user/application data in the resulting token rather than have to look up a session ID in the database on each request. For that we want to use JWT.
+
+JWT (JSON Web Token) is a self-contained, secure and standard way of transmitting data between applications and clients as JSON objects.
```

Comment:
I'd also consider briefly explaining above the fold that using JWTs lets us use a standardized technology to cut our auth workflow down by one round-trip, to motivate the discussion. I think I'd leave the details below, but that starts things off with the assertion that not only is it a cool tech, but we use it to improve performance and reduce application complexity.